index.html

```diff
@@ -56,7 +55,7 @@ <h3>LET'S CONNECT</h3>
         </section>
       </section>
       <section id="my_work">
-        <section id="p1">
+        <!--<section id="p1">
           <img  class="project_image" src="statics/images/Snapshoot_Portfolio.png" alt="snapshoot image project">
           <div class="general">
```

Comment:
[ OPTIONAL ] Pop-up for mobile does not show buttons and the technologies list is overlapping the text. Kindly see the screenshot below and make the pop-up exact same as in the design
![mobile-pop](https://user-images.githubusercontent.com/35504149/136376827-f7f55620-37a3-420f-9103-78b67aa6c164.png)
.